messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessorConfiguration.java

```diff
@@ -84,6 +84,7 @@ private TrackingEventProcessorConfiguration(int numberOfSegments) {
         this.tokenClaimInterval = DEFAULT_TOKEN_CLAIM_INTERVAL;
         this.autoStart = true;
         this.workerTerminationTimeout = DEFAULT_WORKER_TERMINATION_TIMEOUT_MS;
+        this.initialTrackingTokenBuilder = messageSource -> ReplayToken.createReplayToken(messageSource.createTailToken());
```

Comment:
If we want this to be a full replay, the `ReplayToken` should be constructed with a head token as the `tokenAtReset`.
Right now, it suggests the Streaming Processor that it should replay from position `null` to `null`, essentially meaning nothing happens.

Reply:
The single parameter `createReplayToken` accepts the "token at reset" as the first token. It assumes that the "current position" is set to null, which represents the tail token.
I think you intended to do a `ReplayToken.createReplayToken(messageSource.createHeadToken())`  here.